odxtools/nameditemlist.py

```diff
@@ -40,6 +40,7 @@ def append(self, item: T):
         self._list.append(item)
 
         item_name = self._item_to_name_fn(item)
+        assert item_name.isidentifier()
```

Comment:
Please revert this for now, or change it to log a warning

I am getting way too may errors reported by it.

Reply:
ok, done. do happen to know where this still happens to you? (with the dot-to-underscore change, the comparam stuff should be fine now?)

Reply:
I have item names that start with a number...

Reply:
> I have item names that start with a number...

they seem to be invalid IDs, though: if I didn't get this wrong (which is very well possible), `xsd:ID` must start with a letter or an underscore:

http://books.xmlschemata.org/relaxng/ch19-77151.html and https://stackoverflow.com/questions/1631396/what-is-an-xsncname-type-and-when-should-it-be-used